spirv_msl.cpp

```diff
@@ -8373,6 +8378,95 @@ void CompilerMSL::emit_instruction(const Instruction &instruction)
 			SPIRV_CROSS_THROW("Raster order groups require MSL 2.0.");
 		break; // Nothing to do in the body
 
+	case OpRayQueryInitializeKHR:
```

Comment:
OpConvertUToAccelerationStructureKHR is missing, but I suppose that is expected, since Metal does not support BDA ... Either way, this opcode should have a `SPIRV_CROSS_THROW` case.

Reply:
Also not seeing an implementation for `OpRayQueryGetIntersectionInstanceShaderBindingTableRecordOffsetKHR` either.

Reply:
No MSL function corresponding to OpRayQueryGetIntersectionInstanceShaderBindingTableRecordOffsetKHR in the Intersection Query.